src/neuro_relay/intercept_proxy.py

```
@@ -40,16 +40,22 @@ class InterceptProxy:
 
     def __init__(self, cfg: InterceptProxyConfig):
         self.cfg = cfg
-        self._relay_ws: Optional[websockets.WebSocketClientProtocol] = None
+        self._relay_ws: websockets.WebSocketClientProtocol | None = None
         self._relay_lock = asyncio.Lock()
         self._server = None
 
     async def start(self):
         # Start background connection to Intermediary (optional)
         asyncio.create_task(self._ensure_intermediary_connected())
```

Ruff (0.14.8)

Store a reference to the return value of `asyncio.create_task`

(RUF006)